src/App.js

```diff
@@ -98,12 +100,72 @@ function App() {
   function moveToHistory(input) {
     setHistoryItems([...historyItems, ...shoppingCard]);
     setShoppingCard([]);
+    window.confirm(
+      `Sehr geehrte/r ${inputData.name}! Danke für Ihren Kauf! Wir leiten Sie zu dem Kaufübersicht weiter.`
+    );
     navigate("/history");
   }
   function clearHistory() {
     setHistoryItems([]);
   }
+  function feedbackSubmit(user, feedback, elementForFeedback) {
+    const objectExists = allDataItems.find(
```

Comment:
The variable name implies that you have a boolean value here. Either rename the variable or give it a boolean value (using the Array.prototype.some() method for example)

Reply:
already renamed!